chore: add new job offerings (#574)
ory/web: src/markdown/jobs/senior-performance-marketing-manager.mdx

### Overview

- **Position Type:** Full Time
- **Location:** Munich, Germany
- **Components:** Google Analytics, Hubspot, CMS Plattforms

### What You'll Do

* Ownership over the Performance Marketing of Ory.
* Creation of the performance marketing strategy and goals. This includes audience targeting, action plan, budgeting and performance analytics.
* Design and implementation of all performance marketing efforts and campaigns.
* Evaluating and deciding on software and tools for use in performance marketing.
* Setting up and optimizing relevant paid media channels such as Google Ads, LinkedIn Ads and other new advertising platforms relevant to the target group.
* Creative design of ads and landing pages as well as related A/B testing and performance analysis.
* Creating performance dashboards, analytics requirements and market intelligence reports.
* Conducting regular marketing insights sessions for your Ory colleagues.
* Participate in strategy sessions with Ory management.

### What you bring to the table

> * Fluent in written and spoken English. English is our corporate language.

Reviewer: ```suggestion
* Fluent in written and spoken English.
```

* A strong understanding of customer needs and a drive to provide them with sustainable solutions.
* Three to five years of professional experience in digital marketing.
* An excellent understanding of conversion funnels and customer journeys.
* Hands-on experience with marketing software, automations and tools.
* A strong affinity for data and strong analytical skills.
* Strong interest in marketing trends and best practices.
* An understanding of web technologies such as the HTTP protocol, HTML and Javascript.
* Ideally marketing experience in SaaS, B2B or software environments.

### What you can expect at Ory:
* An international, diverse team that is passionate and pioneering in advancing the security of Internet applications.
* At Ory you will be actively involved in shaping and making decisions that will have a significant impact on the company.
* Access to one of the largest open source communities on the Internet of world-class developers and world-changing projects.
* #PayItForward: As a startup with open source roots, providing access to software is important to us. Ory employees can work 10% of their working hours on open source projects of their choice.

> * A very attractive salary and the possibility of Employee Stock Options.

Reviewer: ```suggestion
* An attractive salary and the possibility of Employee Stock Options.
```
